Bugfixes/format spelling

Nightwalker-87 opened the pull request.

Nightwalker-87 committed:
Spelling, grammar & formatting fixes. 15-19

Nightwalker-87 committed:
Formatting fixes. 20 / Removed unnecessary $NON-NLS$ tags.

Nightwalker-87 commented:
Neuauflage mit neuen Tags.

tsandmann commented:
- Die `// TODO Auto-generated catch block` Kommentare würde ich komplett entfernen
- Bitte keine E-Mail Adressen anderer Leute ergänzen ohne deren explizite Zustimmung, schon allein aus Datenschutzgründen
- `Marker-Interface (abstract class)` ist doppelt (abstrakte Klassen heißen in Java Interface)
- Hier ist durch das Vertauschen der Zeilen jetzt der Sinn vom Kommentar kaputt:
 >  new Entry("Per TCP ...", Config.getIcon("tcpbot16"), onAddTcpBot) /*, 
 >  new Checkbox("Per USB (COM) automatisch", noOp).disable().check()*/ );
 >  // Die Checkbox hat nen Haken und ist unveränderbar disabled (ausgegraut).
 >  // Sinn dahinter ist den Benutzer wissen zu lassen, dass ctSim das automatisch macht

 --> sollte man also nicht tauschen
- kein "No-op" bei leeren catch-Blöcken
- ctSim/view/gui/TableOfSpinners.java Zeile 193: sollte `/*` und nicht `/**` sein
- ctSim/view/gui/WorldViewer.java Zeile 253: sollte `/*` und nicht `/**` sein

Nightwalker-87 commented:
> Bitte keine E-Mail Adressen anderer Leute ergänzen ohne deren explizite Zustimmung, schon allein aus Datenschutzgründen

stammte lediglich aus dem mitgelieferten Changelog, war daher keine neue Info - wurde trotzdem wieder entfernt.

Die übrigen Punkte sind größtenteils schon bearbeitet der Rest wird nach Rückmeldung noch ergänzt.

Nightwalker-87 committed:
Adressed change requests.

Removed // TODO Auto-generated catch block comments.
Removed abstract class in relation to Marker-Interface.
Rearraged a comment in MainWinMenuBar.java.
Replaced "No-op" in empty catch-blocks with "egal".
Fix for TableOfSpinners.java L193 & WorldViewer.java L253.

tsandmann commented:
Danke, aber
 >  new Entry("Per TCP ...", Config.getIcon("tcpbot16"), onAddTcpBot) /*, 
 >  new Checkbox("Per USB (COM) automatisch", noOp).disable().check()*/ );
 >  // Die Checkbox hat nen Haken und ist unveränderbar disabled (ausgegraut).
 >  // Sinn dahinter ist den Benutzer wissen zu lassen, dass ctSim das automatisch macht

ist durch das Vertauschen der Zeilen immer noch kaputt bzw. unverständlich.
